deployment/Makefile

```diff
@@ -15,6 +15,15 @@ deploy: run
 	@echo "You can access it on http://localhost:49631"
 	@echo "------------------------------------------------------------------"
 
+clean:
```

Comment:
hello @timlinux 
should we add like this?
clean: kill rm

because these should be called first